examples/code_llama/code_completion_example.py

```diff
@@ -33,6 +33,7 @@ def main(
     enable_azure_content_safety: bool=False, # Enable safety check with Azure content safety api
     enable_sensitive_topics: bool=False, # Enable check for sensitive topics using AuditNLG APIs
     enable_salesforce_content_safety: bool=True, # Enable safety check with Salesforce safety flan t5
+    enable_llamaguard_content_safety: bool=False, # Enable safety check with Llama-Guard
```

Comment:
Looking at the code path examples/code_llama
Do we want to have Llama Guard model for Code Models? CyberSecEvals is a great benchmark to use post fine tuning of any code models and Llama Guard for content moderation.

Reply:
We had to add it here as it has been added to the [`get_saferty_checker`](https://github.com/facebookresearch/llama-recipes/blob/main/src/llama_recipes/inference/safety_utils.py#L209) util, so kind of had to add it to make the code functional. But as this is generally a safety check on the prompts, I kind of think its pretty much the same with other safety packages we have in place already, so that might be just an addition to the mix.